app/views/data_access/retrieval.html.erb

```diff
@@ -14,9 +14,9 @@
   <nav class="data-access-nav">
     <a href="<%=edsc_path "/data/configure?#{@retrieval.source}" %>"><i class="fa fa-arrow-circle-o-left"></i> Back to Data Access Options</a>
   </nav>
-  <header class="data-access-header">
-    <h1>Success!</h1>
-    <p>Your request has been processed.  See below for information on accessing your data.</p>
+  <header class="data-access-header" data-bind="visible: ui.projectList.collectionsToDownload().length == 0 && ui.projectList.collectionOnly().length == 0 && ui.projectList.submittedOrders().length == 0 && ui.projectList.submittedServiceOrders().length == 0 && ui.projectList.collectionLinks().length == 0">
```

Comment:
@mark, @bo,I think this is actually okay - to further clarify what Mark had said, the initial request in the ticket was to just remove the header entirely.  It was useless information because it said 'Success!' no matter what happened.

Removing it was a bit unsightly - when the page was awaiting the state, we just had an empty page, so that was actually no good.  *Something* needed to be there, and so I thought replacing the header with a vague 'Please standby' type message would help inform the user that yes, something was happening and it should be ready sometime soon.

Once the data-access-container has the 'real' status the header should disappear, regardless of what that 'real' status is.  So it doesn't need to be truly complete for it to disappear - just data-access-container needs to be complete in getting filled with information.

So, if it says it's in a submitting state, it's actually fine to hide the header as the data-access-container is clearly stating the status at that point.

At least that was the mindset I used in making this change ;)